docs/modules/ROOT/pages/how-tos/install/cloudscale.adoc

```diff
@@ -0,0 +1,211 @@
+= Installation on Cloudscale
+
+:toc:
+
+[abstract]
+.General Procedure
+--
+. Render install manifests
+
+. Render ignition config
+
+. Provision bootstrap and control plane nodes
+
+. Provision worker and LB nodes
+
+. Synfection
+
+. Post install configuration
+--
+
+.Prerequisites
+* Cloudscale API token
+* `terraform`
+* `mc` https://docs.min.io/docs/minio-client-quickstart-guide.html[Minio client]
+* `jq`
+* Cloned https://github.com/appuio/terraform-openshift4-cloudscale[terraform-openshift4-cloudscale] repository
+
+
+== Cluster Installation
+
+. Create a new Syn cluster: https://control.vshn.net/syn/lieutenantclusters
+
+. Configure input
++
+[source,console]
+----
+export CLOUDSCALE_TOKEN=<cloudscale-token> # From https://control.cloudscale.ch/user/api-tokens
+export CLUSTER_NAME=<cluster-name>
+export BASE_DOMAIN=ocp4-poc.appuio-beta.ch
+export PULL_SECRET=<redhat-pull-secret> # From https://cloud.redhat.com/openshift/install/pull-secret
+----
+
+. Create S3 buckets
++
+[source,console]
+----
+# Use already exiting bucket user
+curl -H "Authorization: Bearer ${CLOUDSCALE_TOKEN}" \
```

Comment:
Do I have to do something with the output? I guess, wetter I use an existing user user or create one, the `AWS_ACCESS_KEY_ID` and `AWS_SECRET_ACCESS_KEY` variables must be set. I can not see the result of this command being used anywhere (e.g. assigned to the variable `response`).